src/NuGet.Clients/NuGet.SolutionRestoreManager/VsSolutionRestoreService.cs

```diff
@@ -34,7 +35,8 @@ namespace NuGet.SolutionRestoreManager
     [Export(typeof(IVsSolutionRestoreService))]
     [Export(typeof(IVsSolutionRestoreService2))]
     [Export(typeof(IVsSolutionRestoreService3))]
-    public sealed class VsSolutionRestoreService : IVsSolutionRestoreService, IVsSolutionRestoreService2, IVsSolutionRestoreService3
+    [Export(typeof(IVsSolutionRestoreService4))]
```

Comment:
Do we need a brokered test here?

Reply:
This is not a brokered service. Unfortunately we're short on tests here, the manual tests + RPS are what tests this.